src/appshell/qml/Preferences/PercussionPreferencesPage.qml

```diff
@@ -0,0 +1,161 @@
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+import QtQuick 2.15
+
+import Muse.Ui 1.0
+import Muse.UiComponents 1.0
+import MuseScore.Preferences 1.0
+
+import "internal"
+
+PreferencesPage {
+    id: root
+
+    BaseSection {
+        id: percussionPanelPreferences
+
+        title: qsTrc("appshell/preferences", "Percussion")
+
+        navigation.section: root.navigationSection
+
+        CheckBox {
+            id: unpitchedSelectedCheckbox
+            width: parent.width
+
+            text: qsTrc("notation", "Open the percussion panel when an unpitched staff is selected")
+
+            navigation.name: "UnpitchedSelectedCheckbox"
+            navigation.panel: percussionPanelPreferences.navigation
+            navigation.row: 0
+
+            onClicked:  {
+                // TODO: configuration - toggle autoShowPercussionPanel
+            }
+        }
+
+        StyledTextLabel {
+            id: padSwapInfo
+
+            width: parent.width
+
+            horizontalAlignment: Text.AlignLeft
+            wrapMode: Text.Wrap
+            text: qsTrc("notation", "When swapping the positions of two drum pads:")
+        }
+
+        RadioButtonGroup {
+            id: radioButtons
+
+            property int navigationRowStart: unpitchedSelectedCheckbox.navigation.row + 1
+            property int navigationRowEnd: radioButtons.navigationRowStart + model.length
+
+            width: parent.width
+            spacing: percussionPanelPreferences.spacing
+
+            orientation: ListView.Vertical
+
+            model: [
+                { text: qsTrc("notation", "Move MIDI notes and keyboard shortcuts with their sounds"), value: true },
+                { text: qsTrc("notation", "Leave MIDI notes and keyboard shortcuts fixed to original pad positions"), value: false }
+            ]
+
+            delegate: Row {
+                width: parent.width
+                spacing: 6
+
+                RoundedRadioButton {
+                    id: radioButton
+
+                    anchors.verticalCenter: parent.verticalCenter
+
+                    navigation.name: modelData.text
+                    navigation.panel: percussionPanelPreferences.navigation
+                    navigation.row: radioButtons.navigationRowStart + model.index
+
+                    // TODO: checked: modelData.value === pad swap preference
+
+                    onToggled: {
+                        // TODO: configuration - change pad swap preference
+                    }
+                }
+
+                //! NOTE: Can't use radioButton.text because it won't wrap
```

Comment:
Perhaps we should improve it and add the ability to specify the wrap mode

Reply:
Agreed  - I'll address this in a separate PR.